tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2008,6 +2003,11 @@ def setUp(self):
             send_packet(self, self.dst_port_id, arpreq_pkt)
```

Comment:
may be we can add some comments about why move the code to below, someone don't know the history may break it again.

Reply:
done